kernelci/db/kernelci_api.py

```diff
@@ -88,6 +88,12 @@ def get_node(self, node_id):
         resp = self._get('/'.join(['node', node_id]))
         return json.loads(resp.text)
 
+    def get_nodes(self, attributes: dict = None):
+        """Get all nodes matching attributes"""
+        resp = self._get('nodes', params=attributes)
+        # Maximum 10 nodes will be returned
+        return resp.json()[:10]
```

Comment:
I think the limit should be an argument of the method, and ideally this should be implemented as pagination in the API. Then we should also have a `skip` argument to get the next "page" of nodes.

Reply:
Implemented pagination in the API https://github.com/kernelci/kernelci-api/pull/164.
`skip` and `limit` arguments can be sent from client scripts using `attributes` dictionary.